nevergrad/parametrization/helpers.py

```diff
@@ -54,3 +58,61 @@ def flatten_parameter(
     if not with_containers:
         flat = {x: y for x, y in flat.items() if not isinstance(y, (core.Dict, core.Constant)) or isinstance(y, choice.BaseChoice)}
     return flat
+
+
+# pylint: disable=too-many-locals
+def split_as_data_parameters(
```

Comment:
@teytaud 
This is what identifies the order in which the Array should be concatenated